sw/host/opentitanlib/src/backend/mod.rs

```diff
@@ -77,7 +77,7 @@ pub fn create(args: &BackendOpts) -> Result<TransportWrapper> {
         ),
         "hyperdebug" => (
             hyperdebug::create::<StandardFlavor>(args)?,
-            Some(Path::new("/__builtin__/opentitan_verilator.json")),
+            Some(Path::new("/__builtin__/opentitan_hyperdebug.json")),
```

Comment:
We do not have to solve right now, but I wonder how we would handle cases such as future FPGA boards using slightly different pin assignments, (or future ASIC boards having additional voltage converters, which could be manipulated using certain signals).  What I am saying is that `--interface hyperdebug` is not sufficient to chose a pin assignment that attaches labels belonging to the "shield".
I realize that it would be inconvenient for now to always have to accompany `--interface hyperdebug` with `--conf opentitan_hyperdebug.json`, that is in part why one of my original suggestions was to specify only `--conf opentitan_hyperdebug.json` explicitly, and let a declaration in that file dictate the transport.
I guess another way of putting this is that this new file should be called `cw310_hyperdebug.json`, and there may be other equally relevant files for hyperdebug in the future.

Reply:
I'm fine with having a `--conf` file inform the tool about which interface to use.  However, the tool should have built-in configs for the most common configurations.

Reply:
I renamed the config file to `hyperdebug_cw310.json`.